packages/web-components/fast-foundation/src/select/select.ts

```diff
@@ -23,7 +23,7 @@ import { SelectPosition } from "./select.options.js";
  * @public
  */
 export type SelectOptions = StartEndOptions & {
-    indicator?: string | SyntheticViewTemplate;
+    openIcon?: string | SyntheticViewTemplate;
```

Comment:
Probably make this non-optional after removing the hard-linked icon from the template.

Reply:
Per Chris's comment on another icon, I agree they shouldn't be required, so I'll leave it here but I've fixed the hard link.